### PR TITLE
Add set/get netkvm params function for netkvm driver in windows

### PR DESCRIPTION
**Pull request by kimi1978wy**
1 Add netkvmco.dll copy path in virtio_win.py
2 Add set/get netkvm params in utils_net.py

Signed-off-by: Yu Wang <wyu@redhat.com>

id:1566941

### CODE REVIEW

**Comment by kimi1978wy** [comment]
(1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.offload_checksum_windows: PASS (290.09 s)

**Comment by xutian** [comment]
LGTM

**Comment by kimi1978wy** [comment]
@pezhang Could you help to review this patch? Thanks a lot

File: virttest/utils_windows/virtio_win.py [hunk 1]
```diff
@@ -104,3 +106,50 @@ def drive_letter_vfd(session):
...
+    netkvmco_path = session.cmd(find_cmd).strip()
+    logging.info("Found netkvmco.dll file at %s" % netkvmco_path)
+    return netkvmco_path
+
+
+def prepare_netkvmco(vm):
+    """
+    Copy the proper netkvmco.dll to driver c:\\, and register it.
+
+    param session: a session to send command lines
```
[the remark this answers is not in the document]
Comment (kimi1978wy) [reply virttest/utils_windows/virtio_win.py]: sorry , my typo , modified. Thanks for reminder

**Comment by pezhang** [comment]
LGTM.

Acked by: Pei Zhang pezhang@redhat.com

**Pull request merged**